src/main/webapp/script.js

```diff
@@ -37,57 +37,57 @@ function cleanseUrl(url) {
   // TODO: Add checks to make this work if video is not first parameter.
   var videoId = url.split("?");
   videoId = (videoId.length > 1) ? videoId[1].split("&")[0] : videoId[0];
-
+ 
   // If param name present, remove it to isolate video Id.
   videoId = videoId.replace("v=", "");
   
   return videoId;
 }
-
+ 
 /**
  * Fetches data and adds to html
  */
 async function getChart() {
   $('form').submit(async function() {
     commentStats = await getYouTubeComments();
     averageScore = commentStats.averageScore;
-    aggregateValues = commentStats.aggregateValues; 
-
+    averageMagnitude = commentStats.averageMagnitude;
+    sentimentBucketList = commentStats.sentimentBucketList; 
+ 
     const CommentSentimentTable = new google.visualization.DataTable();
-    CommentSentimentTable.addColumn('number', 'InclusiveStart');
-    CommentSentimentTable.addColumn('string', 'SentimentRange');
```

Comment:
Since we switched to a map, our backend sorts it in ascending order - would changing from string sorting to sorting by inclusiveStart be something we should change in the java file? Or should I be sorting again in js

Reply:
Good point, I had forgotten that the new list we get from the backend will be sorted.

As long as that list stays sorted, then not sorting on the front-end is okay.